support-frontend/app/controllers/PaperSubscriptionFormController.scala

```diff
@@ -65,6 +65,7 @@ class PaperSubscriptionFormController(
       maybeIdUser,
       testMode,
       priceSummaryServiceProvider.forUser(testMode).getPrices(Paper, promoCodes),
+      maybePromotionCopy = None,
```

Comment:
Given this has a default value set, you don't need to pass this value through.

Reply:
I passed it here, because  otherwise , it is showing me the error 
<img width="400" alt="image" src="https://github.com/guardian/support-frontend/assets/73653255/2068d9bc-1a82-48bd-a672-44f0ea6dd45e">

And if I pass just the  maybePromotionCopy , it is showing me the error 

<img width="400" alt="image" src="https://github.com/guardian/support-frontend/assets/73653255/0131b58e-46ab-4a25-a8eb-0b8720af7236">

Reply:
Ah, OK - this is a nasty problem with using non-named parameters and so order counts. 
You could add `maybePromotionCopy` to the bottom of the parameters list in `subscriptionsCheckout` - or better, we could add the param names in to avoid this in future. 

Let's maybe pic this up outside of this PR.